buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -454,8 +454,12 @@ class BeamModulePlugin implements Plugin<Project> {
     def checkerframework_version = "3.10.0"
     def classgraph_version = "4.8.104"
     def errorprone_version = "2.10.0"
+    // Try to keep gax_version consistent with gax-grpc version in google_cloud_platform_libraries_bom
+    def gax_version = "2.8.1"
```

Comment:
I think that this one will work without the version as the BOM defines a version for it (https://mvnrepository.com/artifact/com.google.cloud/libraries-bom/24.2.0)

Reply:
I did try this without the version, but the URL becomes invalid if the version is not between the `gax-grpc:` and the `:testlib` on line 549. This is the error produced without the version:

![image](https://user-images.githubusercontent.com/8233863/154387064-7ecad690-c61f-40f2-bfd8-0c62edeb4d5c.png)

Let me know if you know of another way!

Reply:
That is ok, thanks for trying